presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -624,12 +624,7 @@ private ConnectorTableMetadata doGetTableMetadata(ConnectorSession session, Sche
 
     private static Optional<String> getCsvSerdeProperty(Table table, String key)
     {
-        return getSerdeProperty(table, key).map(csvSerdeProperty -> {
-            if (csvSerdeProperty.length() > 1) {
-                throw new PrestoException(HIVE_INVALID_METADATA, "Only single character can be set for property: " + key);
-            }
-            return csvSerdeProperty;
-        });
+        return getSerdeProperty(table, key).map(csvSerdeProperty -> String.valueOf(csvSerdeProperty.charAt(0)));
```

Comment:
csvSerdeProperty.substring(0, 1)

(also, can these properties be set to empty string? will we fail here?)

Reply:
For 1, I used a CTAS because I'm not sure if CTAS and CT follow the same code paths in Presto. But I can change to CT + Insert too once you confirm.

2 and 3 make sense, I'll adjust accordingly.

Reply:
Hive doesn't allow empty strings for any of the 3 properties. Checked in 3.1 and 1.1.

Presto also doesn't allow creating and fails with:

```
presto:default> create table test(a varchar) with (format='CSV',csv_quote='');
Query 20200616_212543_00651_k26cy failed: csv_quote must be a single character string, but was: ''
io.prestosql.spi.PrestoException: csv_quote must be a single character string, but was: ''
	at io.prestosql.plugin.hive.HiveTableProperties.getSingleCharacterProperty(HiveTableProperties.java:264)
	at io.prestosql.plugin.hive.HiveMetadata.getEmptyTableProperties(HiveMetadata.java:939)
	at io.prestosql.plugin.hive.HiveMetadata.createTable(HiveMetadata.java:819)
	at io.prestosql.plugin.base.classloader.ClassLoaderSafeConnectorMetadata.createTable(ClassLoaderSafeConnectorMetadata.java:318)
	at io.prestosql.metadata.MetadataManager.createTable(MetadataManager.java:635)
	at io.prestosql.execution.CreateTableTask.internalExecute(CreateTableTask.java:203)
	at io.prestosql.execution.CreateTableTask.execute(CreateTableTask.java:87)
	at io.prestosql.execution.CreateTableTask.execute(CreateTableTask.java:69)
	at io.prestosql.execution.DataDefinitionExecution.start(DataDefinitionExecution.java:167)
	at io.prestosql.execution.SqlQueryManager.createQuery(SqlQueryManager.java:237)
	at io.prestosql.dispatcher.LocalDispatchQuery.lambda$startExecution$7(LocalDispatchQuery.java:132)
	at io.prestosql.$gen.Presto_333____20200615_192640_2.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```